components/dialog/stories/dialog.stories.js

```diff
@@ -65,17 +110,159 @@ export default {
 	},
 };
 
+const ExampleContent = Typography({
+	semantics: "body",
+	size: "m",
+	content: [
+		"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Auctor augue mauris augue neque gravida. Libero volutpat sed ornare arcu. Quisque egestas diam in arcu cursus euismod quis viverra. Posuere ac ut consequat semper viverra nam libero justo laoreet. Enim ut tellus elementum sagittis vitae et leo duis ut. Neque laoreet suspendisse interdum consectetur libero id faucibus nisl."
+	]
+});
+
+const ExampleButtons = [{
+	variant: "secondary",
+	treatment: "outline",
+	label: "Remind me later"
+}, {
+	treatment: "fill",
+	label: "Enable",
+	variant: "accent"
+}];
+
 export const Default = Template.bind({});
 Default.args = {
-	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+	heading: "Disclaimer",
+	showModal: true,
+	buttons: ExampleButtons,
+	content: [
+		() => ExampleContent
+	],
+};
+
+/*
+ * Stories for the MDX "docs" only.
+ */
+export const Small = Template.bind({});
```

Comment:
The `--small` variant styles currently don't display properly. I think if we moved the `--small` selector block in the CSS below the `--medium`, it would work with the cascade better (similar to how we have it set up with some of the other migrated components). 

Additionally, there might be some refactoring that could be possible in the future. If I inspect any of the dialogs, I saw both modifier classes are being added. That seemed odd, but I couldn't quite figure out where things are buggy. 

For instance, on the large variant, I see these classes: `spectrum-Dialog spectrum-Dialog--medium spectrum-Dialog--large ` even though I should probably only see `spectrum-Dialog spectrum-Dialog--large `

Reply:
It looks like this existing template code may be wrong. It's applying the platform scale instead of the size classes:
`[`${rootClass}--${scale}`]: true`. That should be removed now that you have `size`. The platform scale changes things (variables) from the root via `spectrum--medium` and `spectrum--large`.

Reply:
Was I getting the scale (--medium and --large) confused with the dialog classes (which were also --medium and --large)? I got it now though. thank you.